agent/http-server.c

```diff
@@ -70,28 +70,32 @@ hwangsae_http_server_set_recording_dir (HwangsaeHttpServer * server,
 }
 
 static gchar *
-get_file_path (gchar * base_path, gchar * record_id, gchar * ext)
+get_file_path (gchar * base_path, gchar * file_id, gchar * ext)
 {
+  g_autofree gchar *filename = NULL;
   gchar *file_path;
 
-  file_path =
-      g_strdup_printf ("%s/hwangsae-recording-%s-00000.%s", base_path,
-      record_id, ext);
+  filename = g_strdup_printf ("hwangsae-recording-%s.%s", file_id, ext);
```

Comment:
Since we started supporting filename prefix, I think we can't assume filename starts with `hwangsae-recording`. This should be configurable.

Reply:
Well, `hwangsae-recording` is still the hardcoded prefix that `HwangsaeRecorderAgent` uses, so actually we don't have to make it configurable here unless we allow to change the agent's prefix too.

Reply:
Okay. Then let’s merge as it is.